setup.py

```diff
@@ -46,7 +46,7 @@ def exec_file(path_segments, name):
     py_modules=["canonicaljson"],
     description="Canonical JSON",
     install_requires=[
-        "simplejson>=3.6.5",
+        "simplejson>=3.14.0",
```

Comment:
could you add a comment justifying this? (you're right that it is needed btw)

Reply:
Done!